opal/mca/event/external/configure.m4

```diff
@@ -115,11 +116,41 @@ AC_DEFUN([MCA_opal_event_external_CONFIG],[
                               [opal_event_external_support=yes],
                               [opal_event_external_support=no])
 
+           # Check to see if this check failed because it conflicted with LSF's libevent.so
+           # This can happen if LSF's library is in the LDFLAGS envar or default search
+           # path. The 'event_fini' function is only defined in LSF's libevent.so and not
+           # in Libevent's libevent.so
+           AS_IF([test "$opal_event_external_support" = "no"],
+                 [AC_CHECK_LIB([event], [event_fini],
+                               [AC_MSG_WARN([===================================================================])
+                                AC_MSG_WARN([Possible conflicting libevent.so libraries detected on the system.])
+                                AC_MSG_WARN([])
+                                AC_MSG_WARN([LSF provides a libevent.so that is not from Libevent in its])
+                                AC_MSG_WARN([library path. It is possible that you have installed Libevent])
+                                AC_MSG_WARN([on the system, but the linker is picking up the wrong version.])
+                                AC_MSG_WARN([])
+                                AC_MSG_WARN([Configure will continue and attempt to use the 'internal' libevent])
+                                AC_MSG_WARN([instead of the 'external' libevent.])
```

Comment:
This part of the message isn't true if the user specified `--with-event=external`

Reply:
Good point. I'll generalize the text. If they did specify `--with-libevent=external` then this message will print, but it will error out at the bottom of the configure check.

Reply:
Pushed change. Take a look at the new wording.

Reply:
I'll look at the new wording.  You could also just selectively emit this paragraph.